lib/server.js

```diff
@@ -104,6 +88,12 @@ Server.prototype._assertHaveAllTasks = function () {
 Server.prototype._subscribe = function (queueName) {
   debug('_subscribe(' + queueName + ')')
   this.log.trace('ponos.Server: subscribing to ' + queueName)
+  if (!this._tasks[queueName]) {
+    this.log.warn({
+      queueName: queueName
+    }, 'ponos.Server: handler not defined')
+    return
+  }
```

Comment:
can you wrap this and the next `return` block in a `Promise.resolve().then(function () { ... })`? Then it'll keep with the documentation that it is supposed to return a promise.

once you do that, can you expand the unit tests for `_subscribe` to account for this log? you should be able to stub `server.log.warn`, IIRC.